demos/hr_agent/arch_config.yaml

```diff
@@ -0,0 +1,63 @@
+version: v0.1
+listener:
+  address: 127.0.0.1
+  port: 8080 #If you configure port 443, you'll need to update the listener with tls_certificates
+  message_format: huggingface
+
+# Centralized way to manage LLMs, manage keys, retry logic, failover and limits in a central way
+llm_providers:
+  - name: OpenAI
+    provider: openai
+    access_key: OPENAI_API_KEY
```

Comment:
This isn't standard way we should use $OPEN_API_KEY for env replacement

Reply:
we should do sth similar to https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/frontendproxy/Dockerfile#L11

Reply:
filed issue https://github.com/katanemo/arch/issues/213